packages/compiler-vapor/__tests__/basic.test.ts

```diff
@@ -2,7 +2,10 @@ import * as CompilerVapor from '../src'
 // import * as CompilerDOM from '@vue/compiler-dom'
 import { parse, compileScript } from '@vue/compiler-sfc'
 import source from './fixtures/counter.vue?raw'
-
+import singleRootSource from './fixtures/singleRoot.vue?raw'
```

Comment:
The fixtures test is for dev only. Please add unit test separately.
See https://github.com/vuejs/core-vapor/blob/acec2409c7694c5dd37a3f173fe432899549866e/packages/compiler-vapor/__tests__/compile.test.ts